core/src/main/kotlin/in/specmatic/core/pattern/NumberPattern.kt

```diff
@@ -19,7 +19,7 @@ data class NumberPattern(
     val maximum: BigDecimal = HIGHEST_DECIMAL,
     val exclusiveMaximum: Boolean = false,
     override val example: String? = null,
-    val isDoubleFormat: Boolean = true
+    val isDoubleFormat: Boolean = false
```

Comment:
Why was the default changed from true to false?

Reply:
When `isDoubleFormat` was introduced, its value was kept as true by default. 
This default value was working until now since we were **not returning a double value** when min and max were not set. 
Now that we are returning a double value in this scenario, the tests started failing where we were expecting a non-double (Int) value.
Hence, I changed the default value which passed those tests and also maintained the behaviour where if we don't pass the isDoubleFormat value as true, we will get Int value back.